### PR TITLE
Fixed TorpedoStore.java

### PR DESCRIPTION
**Pull request by adamkaposi**

### CODE REVIEW

**Comment by adamkaposi** [comment]
I was able to successfully fix the known 3 bugs in TorpedoStore.java!

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (nmate980829) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Please describe why the argument's value is illegal. This exception message gives little information on how the problem could be resolved.

**Review by nmate980829 — CHANGES_REQUESTED** [review]
The exception message does not contain enough information.

**Review by nmate980829 — APPROVED** [review]
Nice work!

**Pull request merged**